Nataly_Bondarenko/test_framework/tests/test_cart_page.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+
+
+@pytest.mark.regression
+@pytest.mark.cart
+def test_continue_shopping(return_main_page, open_cart_page_with_item):
+    """
+    A method to verify that we are able to continue shopping from the cart page
+    """
+    main_page = return_main_page
+    open_cart_page_with_item.click_on_continue_shopping_button()
+    assert main_page.is_page_logo_displayed() is True, "The `Continue shopping` button doesn't work"
+
+
+@pytest.mark.regression
+@pytest.mark.cart
+def test_move_to_item_page(open_cart_page_with_item, return_item_page):
+    """
+    A method to verify that we are able to move to item page from the cart page
+    """
+    item_page = return_item_page
+    open_cart_page_with_item.go_to_item_page_from_cart()
+    assert item_page.is_item_page_is_displayed() is True, "text"
```

Comment:
what is "text" ?

Reply:
forgat after the test Passed add a normal fail msg. Changed.